etc/ci/performance/README.md

```diff
@@ -0,0 +1,66 @@
+Servo Page Load Time Test
+==============
+
+# Prerequisites
+
+* Python3
+
+# Basic Usage
+
+## Prepare the test runner
+
+* Clone this repo
+* Download [tp5n.zip](http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip), extract it to `page_load_test/`
+* Run `prepare_manifest.sh` to transform the tp5n manifest to our format
+* `python3 -m virtualenv venv; source venv/bin/activate; pip install treeherder-client`
+* Setup your Treeherder client ID and secret as environment variables `TREEHERDER_CLIENT_ID` and `TREEHERDER_CLIENT_SECRET`
+
+## Build Servo
+* Clone the servo repo
+* Compile release build
+* Run `git_log_to_json.sh` in the servo repo, save the output as `revision.json`
+* Put your `servo` binary, `revision.json` and `resources` folder in `etc/ci/performance/servo/`
+
+## Run
+* Run `test_all.sh`
+* Test results are submitted to https://treeherder.mozilla.org/#/jobs?repo=servo
+
+# How it works
+
+* The testcase is from tp5, every testcase will run 20 times, and we take the median.
+* Some of the tests will make Servo run forever, it's disabled right now. See https://github.com/servo/servo/issues/11087
+* Each testcase is a subtest on Perfherder, and their summary time is the geometric mean of all the subtests.
+* Notice that the test is different from the Talos TP5 test we run for Gecko. So you can NOT conclude that Servo is "faster" or "slower" than Gecko from this test.
+
+# Unit tests
+
+You can run all unit tests (include 3rd-party libraries) with `python -m pytest`.
+
+Individual test can be run by `python -m pytest <filename>`:
+
+* `test_runner.py`
+* `test_submit_to_perfherder.py`
+
+# Advanced Usage
+
+## Test Perfherder Locally
+
+If you want to test the data submission code in `submit_to_perfherder.py`, you can setup a local treeherder VM. If you don't need to test `submit_to_perfherder.py`, you can skip this step.
```

Comment:
Is this needed to run `test_submit_to_perfherder.py`? Please add a note either way.